packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -23,6 +23,7 @@
         >
             <card
                 is-rounded
+                has-full-width-footer
```

Comment:
Should this change if checkout is guest or not? There isn't a footer on regular checkout right?

Reply:
No, as this prop just allows to add cardFooter slot, and this slot has a condition on it to be shown only for logout users